pl_bolts/callbacks/variational.py

```diff
@@ -35,15 +46,6 @@ def __init__(
         num_samples: int = 2,
         normalize: bool = True,
     ):
```

Comment:
```suggestion
    ):
        """	
        Args:	
            interpolate_epoch_interval: default 20	
            range_start: default -5	
            range_end: default 5	
            steps: number of step between start and end	
            num_samples: default 2	
            normalize: default True (change image to (0, 1) range)	
        """
```
lets keep it in init